headers/headers.go

```diff
@@ -15,8 +15,10 @@ const (
 	SupportedFeaturesHeaderDelim      = ","
 )
 
+const DEV_VERSION = "0.0.0-DEV"
+
 // Set by GoReleaser using ldflags
-var Version = "DEV"
+var Version = DEV_VERSION
```

Comment:
I changed this because with this definition you can't simply `go run ./cmd/temporal` against a local dev server and have it work. It'll fail on parsing the version info.

If this is an issue for some reason let me know.

Reply:
@feedmeapples

Reply:
yea this change is good with me. We've recently discussed this with @mindaugasrukas . Could you remove the ldflag from makefile as well

Reply:
Done

Reply:
merged here https://github.com/temporalio/cli/pull/202